src/inlines/Context.php

```diff
@@ -41,6 +53,14 @@ public function isHTMLEnabled(): bool {
     return $this->isHtmlEnabled;
   }
 
+  public function areAllURISchemesEnabled(): bool {
+    return $this->areAllURISchemesEnabled;
+  }
+
+  public function getAllowedURISchemes(): keyset<string> {
```

Comment:
this should either be renamed to `getURISchemeAllowList(), or `setURISchemeAllowList()` should be renamed to `setAllowedURISchemes()`- I prefer the latter :)